tools/import-image-tests

```diff
@@ -37,7 +38,7 @@ def list_tests(source) -> Iterator[Tuple[str, str]]:
         yield name, fullpath
 
 
-def load_test(name, fullpath) -> dict:
+def load_test(name, fullpath, filters) -> Optional[dict]:
```

Comment:
If you do the typing note that we need to support 3.6 so we can't use default type subscription (that was a 3.8 (?) feature). Use `Optional[Dict]` and *preferably* also type the `Dict` generic as mypy will error on it in strict mode: `Optional[Dict[str, Any]]`. And honestly, I'd just make it `Dict[str, Any]` and return an empty dict instead of None :)